CHANGELOG.rst

```diff
@@ -204,6 +204,7 @@ IMPORTANT: Python 2 is no longer supported as of this release.  Some packages ma
    * Fix broken links (#1311)
    * Update install instructions (#1238 & #1315)
    * Update New Developers Guide
+   * Update conda install instruction
```

Comment:
We have made the 0.7.0 release, so this changelog entry should now come near the top of the file under the "Next Version" heading.

Also, please include a reference to this pull request number in parentheses.

Reply:
What will be the name of the next version @gonuke? `0.7.1`?

Reply:
That's not clear, but we can leave the header as "Next Version" until that is decided.  I expect it will be 0.7.1, but until we have a release candidate, we'll continue to refer to is as "Next Version"